src/app/logger.service.ts

```diff
@@ -10,11 +11,24 @@ export const LOGGER_DELEGATE = new InjectionToken<Logger>(
 
 @Injectable({ providedIn: "root" })
 export class LoggerService implements Logger {
+  @Output() stateChanged = new Subject();
```

Comment:
`@Output` decorators do not make sense in Services - only in Components and Directives